scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -36,6 +36,10 @@ function doInstallEclipsePlugin() {
     echo
     doEchoAttention "Installation of plugin ${plugin} failed. Please try to install manually or rerun setup (devon eclipse setup)."
     echo
+  else
+    echo
+    doSuccess "Installation of plugin ${plugin} was successful."
```

Comment:
I add this line to get an output if a plugin has been installed successfully.

Reply:
Can you remove the empty `echo` statements?